index.html

```diff
@@ -204,5 +204,66 @@ <h3>About Me</h3>
         <li class="my_skills">Terminal</li>
       </ul>
     </section>
+    <section class="motivation">
+      <p>
+        I'm always<br />
+        interested in hearing <br />
+        about new projects,<br />
+        so if you'd like to chat please get in <br />
+        touch.
+      </p>
+    </section>
+    <section class="form">
+      <form action="https://formspree.io/f/xeqnejbr" method="post">
+        <div>
+          <input
+            type="text"
+            id="full_name"
+            placeholder="Full name"
+            name="Full_name"
+            maxlength="30"
+            required
+          />
+        </div>
+        <div>
+          <input type="email" id="email" placeholder="Email" name="Email" />
+        </div>
+        <div>
+          <textarea
+            id="comment"
+            placeholder="Enter text here"
+            name="Comment"
+            maxlength="500"
+            required
+          ></textarea>
+        </div>
+        <div class="comment_button">
+          <button type="submit">Get in touch</button>
+        </div>
+      </form>
+      <div class="rectangle">
+        <hr class="footer_line" />
+      </div>
+    </section>
```

Comment:
- [ ] Please let's make these sections to be centered using the correct margins and width as in the template so that it follows the same alignment with the rest of the sections and also match your template.

Yours:
<img width="217" alt="Loom_2k3Clvk9Sk" src="https://user-images.githubusercontent.com/81468675/149135602-862e0953-6131-47b1-85a3-b77b5ec04955.png">

Your Template:
<img width="232" alt="Loom_3zV9Ofr5i7" src="https://user-images.githubusercontent.com/81468675/149135630-9f43df2d-6206-48af-80a1-8929a69599ec.png">